app/views/admin/schedules/index.html.haml

```diff
@@ -25,8 +25,21 @@
                 method: :get, class: 'btn btn-primary'
                 = link_to 'Delete', admin_conference_schedule_path(@conference.short_title, schedule.id),
                 method: :delete, class: 'btn btn-danger', data: { confirm: "Do you really want to delete Schedule #{schedule.id}?" }
-
-.row
-  .col-md-12.text-right
-    = link_to 'Add Schedule', admin_conference_schedules_path(@conference.short_title),
-    method: :post, class: 'btn btn-primary'
+- if @rooms.nil?
+  .h3
+    No Venue!
+    %small
+      = link_to 'Create a venue with rooms', new_admin_conference_venue_path
+      before creating the schedule.
+- elsif @rooms.present?
+  .row
+    .col-md-12.text-right
+      = link_to 'Add Schedule', admin_conference_schedules_path(@conference.short_title),
+      method: :post, class: 'btn btn-primary'
+- else
```

Comment:
The logic of the if statement is not very straight forward here. The existence of @rooms implies there is a venue, however when reading the code, it is not explicit what the else means (ie. no venue and no rooms). How could we re-write the if-statement in a way that is easier to read and understand later?